backend/Squirrel.Core/Squirrel.Core.DAL/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Squirrel.Core.DAL.Context;
+
+namespace Squirrel.Core.DAL.Extensions;
+
+public static class ServiceCollectionExtensions
+{
+    public static void AddSquirrelCoreContext(this IServiceCollection services, IConfiguration configuration)
+    {
+        var connectionsString = configuration.GetConnectionString("SquirrelCoreDBConnection");
+        services.AddDbContext<SquirrelCoreContext>(options =>
+            options.UseSqlServer(
```

Comment:
what if we use `Porstgres`?

Reply:
yeah, I think if we use 2 DB in `Console App` probably we should be able to select 2 DB here as well, but it seems you are right